<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drawing::DrawingGanttWidget</class>
 <widget class="QWidget" name="drawing::DrawingGanttWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="drawing::GanttView" name="ganttView"/>
   </item>
  </layout>
  <action name="actSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>drawing::GanttView</class>
   <extends>QGraphicsView</extends>
   <header location="global">src/drawing/ganttview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
